### PR TITLE
Card posting on channel is up and running

### PR DESCRIPTION
**Pull request by allen-tran**
1. Sends embedded message as photo shows:
2022-08-06 at 9 44 30 AM

2. Introduces a random color generator for each card

3. Adds a simple console log to let devs know the server has started

4. Updates README.md with demo and fixed previous typo
5. Updates dev set up doc

### CODE REVIEW

File: disc/cog.py [hunk 1]
```diff
@@ -19,7 +19,25 @@ def __init__(self, bot):
...
-        await ctx.send(f"Title: {data.title}")
+
+        colors = Colors()
+        random_color = colors.random_color()
+        final_color = discord.Colour(random_color)
+
+        description = f"**{data.company}**\n*{data.title}*\n\n"
+        description += f"Location: {data.location}.\n\n"
+        description += f"This job was posted __{data.posted_time_ago}__.\n\n"
+        description += f"{data.summary}\n\nApply Now: {data.url}\n"
```
Comment (allen-tran) [inline disc/cog.py:30]: @truongluan303 I know this looks a little crazy but I changed the description implementation to use string concatenation due to an indentation bug found on mobile devices. See photo below:
IMG_3815

With the string concatenation implementation, the bug fix now looks like this:
IMG_3816

Let me know if you have any feedback on this or suggestions.

Comment (truongluan303) [reply disc/cog.py:30]: We have 2 options to fix this.
You can either line up with the `"""`, like this:
```python
string = """line one
            line two
            line three"""
```
However, IMO, the cleanest way is:
```python
string = (
    "line 1\n"
    "line 2\n"
    "line 3\n"
)
Notice we have no commas
```

File: docs/setup.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Development Setup Instruction
 
-A complete guild to setup your development environment :D
+A complete guide to setup your development environment :D
```
Comment (truongluan303) [inline docs/setup.md:3]: Nice catch!

File: utils/__init__.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
+from random import randint
+
+class Colors:
+    def __init__(self):
+        self.color_list = [0x36a64f, 0x288BA8, 0x746AB0, 0xE83845, 0xFFCE30, 0xE389B9]
+
+    def random_color(self):
```
Comment (truongluan303) [inline utils/__init__.py:7]: Not essential, so feel free to ignore this, but do we actually want random color? I'm thinking about using color for job platforms (e.g LinkedIn vs Glassdoor, etc), or for company level (e.g. Netflix vs a random company).
We can talk more about this later. But I think it's still nice to use this random feature now.

File: README.md [hunk 4]
```diff
@@ -8,7 +8,7 @@ A simple job posting Discord bot that leverages a custom LinkedIn scraping API t
 - simply type:`/post-job <job-url>` into any channel
 
 ## Demo ✨
-- **in progress**
+![ezgif com-gif-maker](https://user-images.githubusercontent.com/63386979/183254555-875f21a0-6943-4fa9-ad80-4f3ce2532d2d.gif)
 
 ## Development Setup
-Please check (the setup doc)[./docs/setup.md] for setup instructions.
+Please check [the setup doc](./docs/setup.md) for setup instructions.
```
Comment (truongluan303) [inline README.md:14]: Nice catch!

**Review by truongluan303 — CHANGES_REQUESTED** [review]
Just the string problem, other than that this looks awesome.

**Comment by truongluan303** [comment]
Not related, but I'm pretty happy we working on stuff together and doing professional PR reviews. Would be nice if we can persuade Shubbie and others to join.

**Comment by allen-tran** [comment]
> Not related, but I'm pretty happy we working on stuff together and doing professional PR reviews LOL. Would be nice if we can persuade Shubbie and others to join.
> 
> 

Yeah bro I agree ❤️ Let's try to get shubbie on some of this dev stuff and itll be even better

File: disc/cog.py (hunk 1 above)
Comment (truongluan303) [inline disc/cog.py:30]:
```suggestion
        description = (
            f"**{data.company}**\n*{data.title}*\n\n"
            f"Location: {data.location}.\n\n"
            f"This job was posted __{data.posted_time_ago}__.\n\n"
            f"{data.summary}\n\nApply Now: {data.url}\n"
        )
```

**Review by truongluan303 — APPROVED** [review]

**Comment by truongluan303** [comment]
Looks like the string trick works!
image

**Pull request merged**